first/frc/index.md

```diff
@@ -13,3 +13,11 @@ This season's game is *FIRST* Infinite Recharge. The teaser and game rules are p
   <a class="btn" href="https://firstfrc.blob.core.windows.net/frc2020/Manual/2020FRCGameSeasonManual.pdf" target="_blank">Download Manual</a>
   <a class="btn" href="http://www.firstinspires.org/resource-library/frc/competition-manual-qa-system?utm_source=hs_email&utm_medium=email&utm_content=40176757&_hsenc=p2ANqtz-9sRtvBregjMA_3wwl7OVcCXZdE7pCduMWXk9_UYkxStV180YhEK_6jamp_zNM9VP7Dety_tN46Pex47KTi8U5wjLgX3Q&_hsmi=40176757" target="_blank">Other Resources</a>
 </div>
```

Comment:
You need to add a line break (`<br />`) after the buttons, otherwise the header below will be too close to them.

Reply:
**Note:** If this section is moved to the end of the page as I suggest in another comment, you don't need a line break after the buttons.